app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -906,9 +924,8 @@ public void displayLoginSkippedWarning() {
     }
 
     private void handleLocationUpdate(fr.free.nrw.commons.location.LatLng latLng, LocationServiceManager.LocationChangeType locationChangeType){
-        setMapBoxPosition(latLng);
```

Comment:
Have we removed this intentionally @neslihanturan ?

Reply:
Hey @ashishkumar468 , yes it was intentional. As recenter method already does a similar effect and I can't recognize any other function of having it since there is no JavaDocs either. Even if we will need it, we should have it inside presenter.updateNearbyMap method, nowhere else. 

Can you think of any issues that will occur with lack of this method?

Reply:
Hey @ashishkumar468 , can you re-view your review:)